components/script/dom/htmlscriptelement.rs

```diff
@@ -184,6 +184,9 @@ pub(crate) struct HTMLScriptElement {
     /// Document of the parser that created this element
     parser_document: Dom<Document>,
 
+    /// Prevents scripts that move between documents during preparation from executing.
```

Comment:
Add a link to https://html.spec.whatwg.org/multipage/scripting.html#preparation-time-document please.